docker-compose.yml

```diff
@@ -1,16 +1,14 @@
 name: movies
 
-endpoints:
-  - service: frontend
-    path: /
-    port: 80
-  - service: api
-    path: /api
-    port: 8080
-
 services:
 
+  movies:
+    build: reverse-proxy
+    ports:
+      - 80:80
+
   frontend:
+    scale: 1
```

Comment:
I would remove this one, it's redundant. and deprecated in the compose spec

Reply:
awee.. its 100% compatible and shows one of our strengths 

also, yes I will remove it